src/deadline/client/cli/_groups/attachment_group.py

```diff
@@ -0,0 +1,125 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+All the `deadline attachment` commands:
+    * upload
+    * download
+"""
+from __future__ import annotations
+
+import click
+import boto3
+
+from typing import Optional
+
+from .._common import _apply_cli_options_to_config, _handle_error
+from ...config import config_file
+from deadline.client.api._attachment import attachment_download as download_attachment_api
+from deadline.client import api
+from deadline.job_attachments._aws.deadline import get_queue
+from deadline.job_attachments.exceptions import MissingJobAttachmentSettingsError
+from deadline.job_attachments.models import JobAttachmentS3Settings
+
+
+@click.group(name="attachment")
+@_handle_error
+def cli_attachment():
+    """
+    Commands to work with AWS Deadline Cloud Job Attachments.
+    """
+
+
+@cli_attachment.command(
+    name="download",
+    help="BETA",
+)
+@click.option(
+    "--manifests",
+    multiple=True,
+    required=True,
+    help="Comma seaparated File paths to manifest formated files.",
+)
+@click.option(
+    "--s3-root-path", help="Job Attachments S3 root path including bucket name and root prefix."
+)
```

Comment:
Just thinking of an alternative, but what we have here is perfectly fine.

How about just taking in an S3 URI? Then we can strip the `s3://` part from it. This is copy/pastable from the S3 console. If we do this, should also consider renaming the option to `--attachments-s3-uri` or something (just `--s3-uri` feels a little too generic IMO)

Reply:
Yes I'm adding `s3://` in the next PR. As for the name of the option, David and I agreed to keep it as is for now, we'll probably get feedback about naming anyways.